lib/poxa/web_hook/dispatcher.ex

```diff
@@ -36,6 +37,7 @@ defmodule Poxa.WebHook.Dispatcher do
   end
 
   defp send_web_hook!(events) do
+    Logger.debug "Sending webhook request."
```

Comment:
Expression produces a value of type 'ok' | {'error',_}, but this value is unmatched